myosuite/envs/myo/myochallenge/__init__.py

```diff
@@ -41,6 +41,26 @@
         }
     )
 
+# Register MyoChallenge Manipulation P2 Evals
+register(id='myoChallengeRelocateP2eval-v0',
+    entry_point='myosuite.envs.myo.myochallenge.relocate_v0:RelocateEnvV0',
+    max_episode_steps=150,
+    kwargs={
+        'model_path': curr_dir+'/../../../simhive/myo_sim/arm/myoarm_object_v0.16(mj237).mjb',
```

Comment:
using mjb instead of xml

Reply:
switched to xml. that one is still used the procedurally generated objects, is that intentional?